roles/openshift_prometheus_operator/tasks/install_prometheus_operator.yaml

```diff
@@ -13,34 +13,28 @@
         config_base: /etc/origin
  
 - name: Copy files to temp directory
-  command: >
-    cp -r /usr/share/ansible/openshift-ansible/roles/openshift_prometheus_operator/files/{{ item }} 
-    {{ mktemp.stdout }}/
-  with_items:
-  - manifests
-  - assets
-  - project.yaml
- 
+  copy:
+    src: "."
+    dest: "{{ mktemp.stdout }}/"
+    directory_mode: yes
+  
+
 - name: Copy admin client config
   command: >
     cp {{ openshift.common.config_base }}/master/admin.kubeconfig {{ mktemp.stdout }}/admin.kubeconfig
   changed_when: false
  
-- name: Ensure monitoring project exists
-  command: >
-    {{ openshift_client_binary }} apply -f "{{ mktemp.stdout }}/project.yaml"
-    --config={{ mktemp.stdout }}/admin.kubeconfig
+# namespace
+- name: Add prometheus project
+  oc_project:
+    state: present
+    name: openshift-monitoring
+    description: Prometheus Operator
```

Comment:
Does this work? `oc new-project` does not allow openshift-* projects. Maybe oc_project uses a more native path. Just wanted to make sure.

Reply:
My concern stems from this code: https://github.com/openshift/openshift-ansible/blob/master/roles/lib_openshift/library/oc_project.py#L1609

Reply:
This ansible role works. I tested it on my cluster.
The prometheus namespace was called `openshift-metrics`, and I also created it using `oc_project`  module.

I didn't get too much into the modules code, so I don't know. It might use a different configuration